datumaro/plugins/synthetic_data/utils.py

```diff
@@ -0,0 +1,198 @@
+# Copyright (C) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import logging as log
+import os
+import os.path as osp
+import warnings
+
+import cv2 as cv
+import numpy as np
+import requests
+
+warnings.filterwarnings("ignore", message=r"(invalid value|overflow) encountered")
```

Comment:
You should narrow this down to a specific module and category, to avoid suppressing warnings in unrelated code.